index.html

```diff
@@ -172,9 +172,16 @@ <h2>
         <p>
           A <dfn>push subscription</dfn> is a message delivery context established between the
           <a>user agent</a> and the <a>push service</a> on behalf of a web application. Each
-          <a>push subscription</a> is associated with a <a>service worker registration</a> and a
+          <a>push subscription</a> is associated with a [=push subscription owner=].
+        </p>
+        <p>
+          A <dfn>push subscription owner</dfn> refers to a <a>service worker registration</a>. A
           <a>service worker registration</a> has at most one <a>push subscription</a>.
         </p>
+        <p class="note">
+          Future updates to this specification could expand the definition of a [=push subscription
+          owner=] to include other types, broadening the range of possible implementations.
```

Comment:
I'd like to see that future update before adding another layer of indirection.  Do you have something more concrete to share?

Reply:
Yes, I can send the larger PR... thought I'd start small and build it up, but happy to just send the larger change that separates between service worker and using `window.navigator.pushManager`.